### PR TITLE
dev: run black on the files

### PR DESCRIPTION
**Pull request by jandom**
run the black formatter on this codebase, only auto-generate code changes

### CODE REVIEW

File: maintainer/update_json_stubs_sitemap.py [hunk 1]
```diff
@@ -23,29 +23,31 @@
...
-URL = os.environ['URL']
-VERSION = os.environ['VERSION']
+URL = os.environ["URL"]
+VERSION = os.environ["VERSION"]
 
 if "http" not in URL:
-    raise ValueError("URL should have the transfer protocol (HTTP/S). "
-                     f"Given: $URL={URL}")
+    raise ValueError(
+        "URL should have the transfer protocol (HTTP/S). " f"Given: $URL={URL}"
```
Comment (lilyminium) [inline maintainer/update_json_stubs_sitemap.py]:
```suggestion
        f"URL should have the transfer protocol (HTTP/S). Given: $URL={URL}"
```

File: maintainer/update_json_stubs_sitemap.py [hunk 2]
```diff
@@ -23,29 +23,31 @@
...
+    )
 
 try:
     int(VERSION[0])
 except ValueError:
-    raise ValueError("$VERSION should start with a number. "
-                     f"Given: $VERSION={VERSION}") from None
+    raise ValueError(
+        "$VERSION should start with a number. " f"Given: $VERSION={VERSION}"
```
Comment (lilyminium) [inline maintainer/update_json_stubs_sitemap.py]:
```suggestion
        f"$VERSION should start with a number. Given: $VERSION={VERSION}"
```

**Review by lilyminium — APPROVED** [review]
Just a couple minor suggestions where combining strings into a single line was kind of weird; otherwise LGTM, thanks @jandom!

**Review by RMeli — APPROVED** [review]
As discussed in https://github.com/MDAnalysis/mdanalysis/issues/2450, `black` is (was?) still somewhat controversial. Since here the changes are applied only to internal scripts, I think this PR would not be too controversial. 

I'm happy to see  approval from @lilyminium on this for the UserGuide, so unless someone else objects I think we can move forward. 

(Maybe applying black to the Jupyter Notebooks, which are user-facing, will be more controversial, but we can discuss if/when that's the case.)

File: doc/source/conf.py [hunk 3]
```diff
@@ -217,4 +226,6 @@ def sort_authors(filename):
 # rst-epilog implements substitutions
 rst_epilog = """
 .. |MDAnalysis_version| replace:: {0}
-""".format(MDAnalysis_version)
+""".format(
+    MDAnalysis_version
+)
```
Comment (RMeli) [inline doc/source/conf.py]: This is one of the weird ones, but harmless. And would be a non-issue with f-strings.

Maybe a first good step to apply black to the main code base would be to convert `format` statements into f-strings.

Comment (jandom) [reply doc/source/conf.py]: Good shout, I turned this into an f-string

**Pull request merged**